src/stacks-hierarchy/Root_PurchaseOverviewScreen/components/PurchaseMessage.tsx

```diff
@@ -0,0 +1,15 @@
+import {MessageInfoBox} from '@atb/components/message-info-box';
+import React from 'react';
+import {useOtherDeviceIsInspectableWarning} from '@atb/modules/fare-contracts';
+
+export const PurchaseMessage = () => {
```

Comment:
Could this get a more specific name now? Maybe even this wrapper component isn't necessary?